extensions/intellij/intellij_generator_plugin/src/main/java/com/bloc/intellij_generator_plugin/language_server/BlocLanguageServerFactory.kt

```diff
@@ -63,4 +71,31 @@ class BlocLanguageServerFactory : LanguageServerFactory {
             .createNotification(content, type)
             .notify(project)
     }
+
+    override fun isEnabled(project: Project): Boolean {
+        val blocToolsVersion = getBlocToolsVersion()
```

Comment:
Looks like this method is being called many times and results in a really laggy user experience so we should probably rework this.